lib/internal/event_target.js

```diff
@@ -343,7 +343,7 @@ class EventTarget {
     // TODO(@jasnell): If it's determined this cannot be backported
     // to 12.x, then this can be simplified to:
     //   const capture = Boolean(options?.capture);
-    const capture = options != null && options.capture === true;
+    const capture = options?.capture === true;
```

Comment:
We should remove the TODO if we're doing that.

Reply:
Nice catch, the Todo here has been removed